tests/test_tasks.py

```diff
@@ -40,6 +40,13 @@ def test_from_dict(self):
         self.assertEqual(input_schema, task.input_schema)
         self.assertEqual(label_schema, task.label_schema)
 
+    def test_value_error_unique_labels(self):
+        with self.assertRaises(ValueError):
+            # Add duplicate labels
+            self.labels.append("neg")
+            task = TextClassification(text_column="input_text", label_column="input_label", labels=self.labels)
```

Comment:
Here you add "neg" to self.labels which is used in the other tests `test_column_mapping`, `test_from_dict`.
To not interfere with the other tests you can just copy the labels list:

```suggestion
            labels = self.labels + self.labels[:1]
            task = TextClassification(text_column="input_text", label_column="input_label", labels=labels)
```

Reply:
Yes, that's exactly what I meant! More readable and avoids unintended behavior.